var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -84,6 +87,8 @@ class PyScipy(PythonPackage):
 
     # NOTE: scipy picks up Blas/Lapack from numpy, see
     # http://www.scipy.org/scipylib/building/linux.html#step-4-build-numpy-1-5-0
```

Comment:
Can you remove these two lines above? The link is broken, the comment is incorrect.